src/index.js

```diff
@@ -62,7 +62,10 @@ const methods = loggerRef => {
     });
   };
 
-  const logifyAll = (obj, promisified = false) => {
+  const logifyAll = (obj, promisified) => {
+    if (promisified === undefined) {
+      promisified = '';
+    }
     if (promisified === true) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this "===" check; it will always be false. Did you mean to use "=="? [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS3403)